tasks/apt_source_deb822.yml

```diff
@@ -0,0 +1,17 @@
+# SPDX-FileCopyrightText: 2025 Stefan Haun <tux@netz39.de>
+# SPDX-License-Identifier: MIT
+---
+
+- name: Add deb822-style Docker's APT repository
+  ansible.builtin.template:
+    src: templates/docker.sources.j2
+    dest: /etc/apt/sources.list.d/docker.sources
+    owner: root
+    group: root
+    mode: '0644'
+  register: apt_repo
```

Comment:
Would it be possible to use Ansible module 'ansible.builtin.deb822_repository' instead? (New in ansible-core 2.15, post Debian-12)  We would not need the handcrafted template anymore then.